docs/content/reference/exploiters/SNMP.md

```diff
@@ -14,4 +14,4 @@ If a community is configured with read-write access, the machine is vulnerable t
 
 In versions 1 and 2c of the SNMP protocol the only means of authentication is the community string. In these versions of SNMP, community strings are sent as plaintext in all requests. Version 3 adds user-based authentication and encryption to the SNMP protocol.
 
-This exploiter uses version 2c of the SNMP protocol. It brute-forces machines by using usernames and passwords provided by the user (see [configuration]({{< ref "/usage/configuration" >}}) for instructions) as SNMP community strings.
+This exploiter uses version 2c of the SNMP protocol. It brute-forces machines by using usernames and passwords provided by the user (see [configuration](/usage/configuration) for instructions) as SNMP community strings.
```

Comment:
[Same comment as on SMBExec](https://github.com/guardicore/monkey/pull/4198/files#r1638318764). Looks like SSHexec and WMIExec link to this page as well

Reply:
Same as above.